examples/python/src/hello-world.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import os
+from posixpath import join
+
+import requests
+from flask import Flask, request
+
+logging.basicConfig(level=logging.DEBUG)
+
+app = Flask(__name__, static_url_path='/static', static_folder='./static')
+
+DEFAULT_APP_PORT = 8080
+DEFAULT_APP_HOST = 'localhost'
```

Comment:
For ease of configurability, maybe we make this `0.0.0.0`? Also let's make the env var be `HOST`

Reply:
done